starter-code/src/functions-and-arrays.js

```diff
@@ -1,5 +1,11 @@
 // Find the maximum
 
+function maxOfTwoNumbers(one, two) {
+  if(one > two) return one;
+  if(two > one) return two;
+  if(one == two) return one;
```

Comment:
You don't need three conditions here. If the first number is not greater than the second one, you can return the second one, because either the second one is greater or they are equal (and then doesn't matter wich number you return):
```
function maxOfTwoNumbers(one, two) {
  if (one > two) {
    return one;
  } else {
    return two;
  }
}
```
Or, if you prefer it:
```
function maxOfTwoNumbers(one, two) {
  if (one > two) return one;
  return two;
}
```